packages/next/client/index.tsx

```diff
@@ -763,10 +763,6 @@ function doRender(input: RenderRouteInfo): Promise<any> {
       ).forEach((el) => {
         el.parentNode!.removeChild(el)
       })
-
-      // Force browser to recompute layout, which should prevent a flash of
-      // unstyled content:
-      getComputedStyle(document.body, 'height')
```

Comment:
I still believe we need to force recompute here, or we should at least be very sure we want to get rid of this. It might be introducing some other form of main thread blocking that has an effect.

Can we ship this as `getComputedStyle(document.body).height` for now?

Reply:
Why changing behavior? I'm pretty sure in all browsers this is a no-op right now.